Pokedex/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by Стефан  Миливоевич on 5.10.22..
+//
+
+import UIKit
+
+class MainScreen: UIViewController, UITableViewDelegate, UITableViewDataSource  {
+   
+    
+    
+    var data = [String()]
+    let titleText = UILabel()
+    let table = UITableView()
```

Comment:
Стол?)))
Может так лучше?
```suggestion
    let tableView = UITableView()
```